src/assessment/builder.py

```diff
@@ -50,11 +50,13 @@
                 </html>
                """
 
+CONFIRMATION_PROMPT = "\nDo you need to modify any of the values for this campaign?"
+
 
 def set_time_zone():
     """Select a timezone from a list of US-based time zones.
 
-    :return: Time zone name based on pytz.
+    : return Time zone name based on pytz.
```

Comment:
Are you using [PyCharm](https://www.jetbrains.com/pycharm/)?  I believe that is where the `:return:` syntax came from in this docstring.  I don't know that we need it- I'm totally ok with doing it like this:
```suggestion
    Returns: Time zone name based on pytz.
```

Whatever is decided, please make sure it is applied consistently throughout this project.

Reply:
Thanks for the reviews and feedback @jsf9k and @dav3r!!

So... I'm not using Pycharm and have used a highly modded version of vim/ale as my IDE for a long time now and always come back to it when trying others. Regarding the `:return:` format, that was just already in place and formatted that way, but my linter didn't like the colon/spacing  combination so I updated to satisfy the linter, forgetting to go back and check the others for consistency. 

I fully agree that it should be consistent though. I just didn't include the work in the scope of this PR. I also prefer your suggested use of `Returns: ` over the original. If there are no objections, I'll go ahead and update it and make sure it's consistent across this file and also create an issue to track the work for circling back and getting the rest of the project updated and consistent.

Reply:
Sounds good to me - thanks!